projectName-web/src/main/webapp/WEB-INF/web.xml

```diff
@@ -17,6 +16,10 @@
         </param-value>
     </context-param>
 
+    <listener>
+        <listener-class>org.terasoluna.gfw.web.logging.HttpSessionEventLoggingListener</listener-class>
```

Comment:
This line is not added.  It replaced definition order.

[Before]
* `ContextLoaderListener`
* `HttpSessionEventLoggingListener`
* `context-param` for `ContextLoaderListener`

[After]
* `ContextLoaderListener`
* `context-param` for `ContextLoaderListener`
* `HttpSessionEventLoggingListener`